goimports/runner.bash.template

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
+set -euo pipefail
+
 GOIMPORTS_SHORT_PATH=@@GOIMPORTS_SHORT_PATH@@
 ARGS=@@ARGS@@
+PREFIX_DIR_PATH=@@PREFIX_DIR_PATH@@
+PREFIX_BASE_NAME=@@PREFIX_BASE_NAME@@
+
+mkdir -p "src/$PREFIX_DIR_PATH"
+ln -shf "$BUILD_WORKSPACE_DIRECTORY" "src/$PREFIX_DIR_PATH/$PREFIX_BASE_NAME"
+
+GOPATH=$(pwd -P)
 
 goimports_short_path=$(readlink "$GOIMPORTS_SHORT_PATH")
-cd "$BUILD_WORKSPACE_DIRECTORY"
-"$goimports_short_path" "${ARGS[@]}" $(find . -type f -name '*.go' @@EXCLUDE_PATHS@@ @@EXCLUDE_FILES@@)
+CURDIR="$GOPATH/src/$PREFIX_DIR_PATH/$PREFIX_BASE_NAME"
+cd "$CURDIR"
+env -i GOPATH="$GOPATH" PATH="$PATH" PWD="$CURDIR" "$goimports_short_path" "${ARGS[@]}" $(find . -type f -name  '*.go' -not -path './bazel-*' @@EXCLUDE_PATHS@@ @@EXCLUDE_FILES@@)
```

Comment:
This runs goimports with only the explicitly provided environment variables to avoid any surprises like `GOROOT` or anything else goimports may depend on today or in the future.